app/src/main/java/no/usn/mob3000/domain/viewmodel/AuthenticationViewModel.kt

```diff
@@ -12,17 +12,19 @@ import no.usn.mob3000.domain.model.AuthError
 import no.usn.mob3000.domain.model.User
 import no.usn.mob3000.domain.usecase.auth.LoginUseCase
 import no.usn.mob3000.domain.usecase.auth.LogoutUseCase
+import no.usn.mob3000.domain.usecase.auth.RegisterUseCase
 
 /**
  * ViewModel to user state and, if necessary, authentication state.
  *
  * @property loginUseCase The Android Use Case handling login business logic.
- * @author frigvid
+ * @author frigvid, Anarox
  * @created 2024-10-21
```

Comment:
Registration is unique enough that it should have its own view model, e.g. `RegistrationViewModel`.